src/psyclone/psyGen.py

```diff
@@ -61,7 +61,7 @@ def colored(text, _):
         :type text: string
         :param _: Fake argument, only required to match interface
                   provided by termcolor.colored
-        :return: The supplied text, unchanged
+        :returns: The supplied text, unchanged
```

Comment:
Either `return` or `returns` is fine.

Reply:
Sphinx user manual has only examples of `returns`.